diff --git a/Components/Input.qml b/Components/Input.qml
index 95bc96f..9a90870 100644
--- a/Components/Input.qml
+++ b/Components/Input.qml
@@ -20,7 +20,7 @@
 import QtQuick 2.11
 import QtQuick.Layouts 1.11
 import QtQuick.Controls 2.4
-import QtGraphicalEffects 1.0
+//import QtGraphicalEffects 1.0
 
 Column {
     id: inputContainer
@@ -72,17 +72,20 @@ Column {
             }
 
             indicator: Button {
-                    id: usernameIcon
-                    width: selectUser.height * 0.8
-                    height: parent.height
-                    anchors.left: parent.left
-                    anchors.verticalCenter: parent.verticalCenter
-                    anchors.leftMargin: selectUser.height * 0.125
-                    icon.height: parent.height * 0.25
-                    icon.width: parent.height * 0.25
-                    enabled: false
-                    icon.color: root.palette.text
-                    icon.source: Qt.resolvedUrl("../Assets/User.svgz")
+                id: usernameIcon
+                width: selectUser.height * 0.8
+                height: parent.height
+                anchors.left: parent.left
+                anchors.verticalCenter: parent.verticalCenter
+                anchors.leftMargin: selectUser.height * 0.125
+                icon.height: parent.height * 0.25
+                icon.width: parent.height * 0.25
+                enabled: false
+                icon.color: root.palette.text
+                icon.source: Qt.resolvedUrl("../Assets/User.svgz")
+                background: Rectangle {
+                    color: "transparent"
+                }
             }
 
             background: Rectangle {
@@ -109,15 +112,6 @@ Column {
                     radius: config.RoundCorners / 2
                     color: "#444"
                     layer.enabled: true
-                    layer.effect: DropShadow {
-                        transparentBorder: true
-                        horizontalOffset: 0
-                        verticalOffset: 0
-                        radius: 100
-                        samples: 201
-                        cached: true
-                        color: "#88000000"
-                    }
                 }
 
                 enter: Transition {
@@ -453,7 +447,7 @@ Column {
 
             contentItem: Text {
                 text: parent.text
-                color: "#444"
+                color: config.Foreground
                 font.pointSize: root.font.pointSize
                 horizontalAlignment: Text.AlignHCenter
                 verticalAlignment: Text.AlignVCenter
@@ -462,8 +456,8 @@ Column {
 
             background: Rectangle {
                 id: buttonBackground
-                color: "white"
-                opacity: 0.2
+                color: config.CurrentLine
+                opacity: 0.5
                 radius: config.RoundCorners || 0
             }
 
@@ -492,7 +486,7 @@ Column {
                     PropertyChanges {
                         target: loginButton.contentItem
                         opacity: 1
-                        color: "#444"
+                        color: config.Foreground
                     }
                 },
                 State {
diff --git a/Components/SessionButton.qml b/Components/SessionButton.qml
index 037d4c1..621da7b 100644
--- a/Components/SessionButton.qml
+++ b/Components/SessionButton.qml
@@ -19,7 +19,7 @@
 
 import QtQuick 2.11
 import QtQuick.Controls 2.4
-import QtGraphicalEffects 1.0
+//import QtGraphicalEffects 1.0
 
 Item {
     id: sessionButton
@@ -46,13 +46,13 @@ Item {
             contentItem: Text {
                 text: model.name
                 font.pointSize: root.font.pointSize * 0.8
-                color: selectSession.highlightedIndex === index ? "#444" : root.palette.highlight
+                color: selectSession.highlightedIndex === index ? config.Foreground : root.palette.highlight
                 verticalAlignment: Text.AlignVCenter
                 horizontalAlignment: Text.AlignHCenter
             }
             highlighted: parent.highlightedIndex === index
             background: Rectangle {
-                color: selectSession.highlightedIndex === index ? root.palette.highlight : "transparent"
+                color: selectSession.highlightedIndex === index ? config.Selection : "transparent"
             }
         }
 
@@ -99,17 +99,8 @@ Item {
 
             background: Rectangle {
                 radius: config.RoundCorners / 2
-                color: "#444"
+                color: config.PanelColor
                 layer.enabled: true
-                layer.effect: DropShadow {
-                    transparentBorder: true
-                    horizontalOffset: 0
-                    verticalOffset: 0
-                    radius: 100
-                    samples: 201
-                    cached: true
-                    color: "#88000000"
-                }
             }
 
             enter: Transition {
diff --git a/Components/UserList.qml b/Components/UserList.qml
index 1fa7170..0a3d059 100644
--- a/Components/UserList.qml
+++ b/Components/UserList.qml
@@ -19,7 +19,7 @@
 
 import QtQuick 2.11
 import QtQuick.Controls 2.4
-import QtGraphicalEffects 1.0
+//import QtGraphicalEffects 1.0
 
 Item {
     id: usernameField
@@ -103,15 +103,6 @@ Item {
                 radius: 10
                 color: root.palette.window
                 layer.enabled: true
-                layer.effect: DropShadow {
-                    transparentBorder: true
-                    horizontalOffset: 0
-                    verticalOffset: 0
-                    radius: 100
-                    samples: 201
-                    cached: true
-                    color: "#88000000"
-                }
             }
 
             enter: Transition {
@@ -194,4 +185,4 @@ Item {
         ]
     }
 
-}
\ No newline at end of file
+}
diff --git a/Main.qml b/Main.qml
index 770f906..0e0e2ef 100644
--- a/Main.qml
+++ b/Main.qml
@@ -20,7 +20,7 @@
 import QtQuick 2.11
 import QtQuick.Layouts 1.11
 import QtQuick.Controls 2.4
-import QtGraphicalEffects 1.0
+//import QtGraphicalEffects 1.0
 import "Components"
 
 Pane {
@@ -37,7 +37,7 @@ Pane {
     palette.highlight: config.AccentColor
     palette.text: config.MainColor
     palette.buttonText: config.MainColor
-    palette.window: "#444"
+    palette.window: config.PanelColor
 
     font.family: config.Font
     font.pointSize: config.FontSize !== "" ? config.FontSize : parseInt(height / 80)
@@ -74,7 +74,7 @@ Pane {
             id: formBackground
             anchors.fill: form
             anchors.centerIn: form
-            color: "#444"
+            color: config.PanelColor
             opacity: config.PartialBlur == "true" ? 0.3 : 1
             z: 1
         }
diff --git a/theme.conf b/theme.conf
index 52658f5..946061e 100644
--- a/theme.conf
+++ b/theme.conf
@@ -6,18 +6,21 @@ Background="Background.jpg"
 ScaleImageCropped=true
 ## Whether the image should be cropped when scaled proportionally. Setting this to false will fit the whole image when scaled, possibly leaving white space. This can be exploited beautifully with illustrations. (Try it with "Illustration.svg")
 
-ScreenWidth=1440
-ScreenHeight=900
+ScreenWidth=1920
+ScreenHeight=1080
 ## Adjust to your resolution to help SDDM speed up on calculations
 
 
 
 ## [Design Customizations]
+CurrentLine="#44475a"
+Selection="#44475a"
+Foreground="#f8f8f2"
 
-MainColor="navajowhite"
+PanelColor="#282a36"
+MainColor="#bd93f9"
 ## Used for all elements when not focused/hovered etc. Usually the best effect is achieved by having this either white or a very light color
-
-AccentColor="white"
+AccentColor="#ff79c6"
 ## Used for elements in focus/hover/pressed. Should be contrasting to the background and the MainColor for best effect
 
 RoundCorners=20
@@ -58,7 +61,7 @@ ForceLastUser=true
 ForcePasswordFocus=true
 ## Give automatic focus to the password field. Together with ForceLastUser this makes for the fastest login experience. 
 
-ForceHideCompletePassword=false
+ForceHideCompletePassword=true
 ## If you don't like to see any character at all not even while being entered set this to true.
 
 ForceHideVirtualKeyboardButton="false"
